doc/source/cluster/kubernetes/user-guides/observability.md

```diff
@@ -21,13 +21,42 @@ kubectl logs $KUBERAY_OPERATOR_POD -n $YOUR_NAMESPACE | tee operator-log
 
 Use this command to redirect the operator's logs to a file called `operator-log`. Then search for errors in the file.
 
-### Method 2: Check custom resource status
+### Method 2: Check the status and events of custom resources.
 
 ```bash
 kubectl describe [raycluster|rayjob|rayservice] $CUSTOM_RESOURCE_NAME -n $YOUR_NAMESPACE
 ```
 
-After running this command, check the status and events of the custom resource for any errors.
+After running this command, check events and the `state`, and `conditions` in the status of the custom resource for any errors and progresses.
+
+
+#### RayCluster `.Status.State`
+
+The `.Status.State` field, which currently represents the cluster's situation, will be deprecated in the future due to its limited representation. Please use the new `Status.Conditions` as an alternative.
+
+| State     | Description                                                                                                                            |
+|-----------|----------------------------------------------------------------------------------------------------------------------------------------|
+| Ready     | The state will be set to `Ready` once all the Pods in the cluster are ready. The `State` will remain `Ready` until the cluster is suspended. |
+| Suspended | The state will be set to `Suspended` when `Spec.Suspend` is set to true and all Pods in the cluster have been deleted.                             |
+
+
+
+#### RayCluster `.Status.Conditions`
```

Comment:
I remember that `.Status.Conditions` has a feature flag. Can you explain how to enable this feature?

Reply:
Might help also add a warning header here that it's an alpha feature

Reply:
Both added.

<img width="1260" alt="image" src="https://github.com/user-attachments/assets/d9c9553e-60a5-4472-b623-f2ba4634c490">